rmw_cyclonedds_cpp/src/rmw_node.cpp

```diff
@@ -3685,10 +3685,17 @@ extern "C" rmw_ret_t rmw_send_request(
   int64_t * sequence_id)
 {
   static std::atomic_uint next_request_id;
-  RET_NULL(client);
-  RET_WRONG_IMPLID(client);
-  RET_NULL(ros_request);
-  RET_NULL(sequence_id);
+  RMW_CHECK_ARGUMENT_FOR_NULL(client, RMW_RET_INVALID_ARGUMENT);
+
+  RMW_CHECK_TYPE_IDENTIFIERS_MATCH(
+    client handle,
+    client->implementation_identifier, eclipse_cyclonedds_identifier,
+    return RMW_RET_INCORRECT_RMW_IMPLEMENTATION);
+
+  RMW_CHECK_ARGUMENT_FOR_NULL(ros_request, RMW_RET_INVALID_ARGUMENT);
+
```

Comment:
@Lobotuerk nit: remove the above three blank lines.

Reply:
Updated on 616b200